README.md

```diff
@@ -2,10 +2,10 @@
 
 This repository contains a reference AWS Platform
 [Configuration](https://crossplane.io/docs/v1.9/getting-started/create-configuration.html)
-for use as a starting point in [Upbound Cloud](https://upbound.io) or
+for use as a starting point in [Universal Marketplace](https://marketplace.upbound.io/) and
```

Comment:
Is it possibly misleading to say "for use" in Universal Marketplace? You're not really using it in there, merely accessing it from there right?